Практика №4 на проверку

AlinaShvetsova opened the pull request.

AlinaShvetsova committed:
Add files via upload

stanislavlyalin commented on pract-04/k_means.py:
```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+# евклидово расстояние между двумя точками
+def dist(A, B):
```
Попробуйте в качестве эксперимента реализовать расстояние городских кварталов. Запустите код и опишите результаты в комментарии к pull request

stanislavlyalin commented on pract-04/k_means.py:
```diff
@@ -0,0 +1,42 @@
+        centers[i,:] = np.mean(sub_X, axis=0)
+    # приписываем каждую точку к заданному классу
+    curr_iteration = class_of_each_point(X, centers)
+    if np.all(curr_iteration == prev_iteration):
```
Попробуйте использовать это условие в цикле while (строка 31). Это упростит код

Veronika2125 committed:
Update k_means.py

Расстояние городских кварталов - это расстояние между двумя точками.
В результате получаем сумму модулей разностей координат между двумя точками.
